mind_matter_api/api/users.py

```
@@ -20,6 +20,8 @@ def init_user_routes(app):
     Initialize user-related API routes.
     """
     @app.route("/users", methods=["GET"])
+    @require_auth
+    @require_admin
```

Ruff (0.8.2)

Undefined name `require_admin`

(F821)